core/state_processor_rip7560.go

```diff
@@ -304,6 +356,7 @@ func prepareAccountValidationMessage(baseTx *types.Transaction, chainConfig *par
 	return &Message{
 		From:              chainConfig.EntryPointAddress,
 		To:                tx.Sender,
+		Nonce:             tx.Nonce,
```

Comment:
Why do you need `Nonce` here? This `Message` is only a part of a transaction and nonce is not checked in it, right?

Reply:
right. leftover when I thought we can use the existing nonce check. removing.